blog/2022-09-12-psp-deprecation.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: KubeSlice Security and PSP Removal
+description: Blog post about KubeSlice Security and PSP Removal
+slug: kubeslice-security-psp-removal
+authors:
+  - name: Leon Nunes
+    url: https://github.com/afro-coder
+hide_table_of_contents: false
+---
+
+In Kubeslice security is something that is Step 0 rather than an afterthought.
+
+#### What are Pod Security Policies?
+> PodSecurityPolicy is a built-in [admission controller](https://kubernetes.io/blog/2019/03/21/a-guide-to-kubernetes-admission-controllers/) that allows a cluster administrator to control security-sensitive aspects of the Pod specification.
+One or more PodSecurityPolicy resources are created in a cluster to define the requirements Pods must meet. Then, RBAC rules are created to control which PodSecurityPolicy applies to a given pod. If a pod meets the requirements of its PSP, it will be admitted to the cluster as usual.
+
+**Note**: The [Kubernetes 1.25 release](https://kubernetes.io/blog/2022/08/04/upcoming-changes-in-kubernetes-1-25/) will remove PSP so if you rely on PSP you should follow the instructions for [migration to Pod Security Admission](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/)
+
+### How KubeSlice implements Security and PSS?
+KubeSlice segregates the Network in Slices as tenants shouldn't need to communicate with all the resources in the cluster.
+A slice is a collection of namespaces, network, or applications over clusters or over different clouds.
+With the creation of such an entity, the effective blast radius during a security incident is reduced.
+
+It also prevents the problem of noisy/nosey neighbours by providing adequate isolation.
+
+Kubernetes defines a standard called [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/)(PSS),
+>The Pod Security Standards define three different policies to broadly cover the security spectrum. These policies are cumulative and range from highly-permissive to highly-restrictive.
+
+This guide outlines the requirements of each policy.
+| Profile | Description
+| -------- | -------- |
+| Privileged | Unrestricted policy, providing the widest possible level of permissions. This policy allows for known privilege escalations.     |
+|Baseline|Minimally restrictive policy which prevents known privilege escalations. Allows the default (minimally specified) Pod configuration.|
+|Restricted|Heavily restricted policy, following current Pod hardening best practices.|
+
+These are Cluster and namespace specific.
+
+However, since Kubeslice creates slices which can span over clusters one can create a secure slice
+by setting the Pod Security Standards(PSS) for a set of namespaces.
+These namespaces will then reside inside a slice.
```

Comment:
This is an obvious thing because you're saying that slice = set of namespaces. Would we be able to insert a code snippet here illustrating it instead? It can be from the documentation specifying what you've said.